server/src/test/java/org/elasticsearch/recovery/RelocationIT.java

```diff
@@ -110,6 +110,13 @@ protected void beforeIndexDeletion() throws Exception {
         internalCluster().assertSameDocIdsOnShards();
     }
 
+    @Override
+    public Settings indexSettings() {
+        return Settings.builder().put(super.indexSettings())
+            // sync global checkpoint quickly so we can verify seq_no_stats aligned between all copies after tests.
+            .put(IndexService.GLOBAL_CHECKPOINT_SYNC_INTERVAL_SETTING.getKey(), "1s").build();
```

Comment:
some tests in this class explicitly set this to `100ms`. Is that not needed anymore now?